editor/src/components/canvas/dom-walker.ts

```diff
@@ -177,7 +179,11 @@ export function useDomWalker(props: CanvasContainerProps): React.Ref<HTMLDivElem
         )
       }
 
-      function getComputedStyle(element: HTMLElement): ComputedStyle {
+      function getComputedStyle(element: HTMLElement, path: TemplatePath): ComputedStyle | null {
+        if (selectedViews.find((sv) => TP.pathsEqual(sv, path)) == null) {
```

Comment:
If you enclose the main bulk with this condition and then `else` the `return null`, then this condition will be a little clearer I think.
```suggestion
        if (selectedViews.find((sv) => TP.pathsEqual(sv, path)) == null) {
        if (selectedViews.some((sv) => TP.pathsEqual(sv, path))) {
```

Reply:
Hmmm yeah it definitely took me a few reads to understand what was going on here. Alternatively if you didn't want to do that you could pull out the condition:
```const isSelected = selectedViews.some((sv) => TP.pathsEqual(sv, path))```

Reply:
fixed

Reply:
@seanparsons in general I prefer an early return of null as opposed to "the main body of the function wrapped in `if`s" style. I think it helps clear out the edge cases at the top so you can focus on the "meat" of the function